helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -637,35 +641,23 @@ private RollingUpdateOp rollingUpdateTimedoutError(final RollingUpdateOpFactory
 
     if (taskStatus.getThrottled().equals(ThrottleState.IMAGE_MISSING)) {
       return opFactory.error(
-              "timed out waiting for job to reach RUNNING due to missing Docker image " +
-                      String.format("(previous job states: %s)", previousJobStatesString),
+              baseError + "due to missing Docker image " + stateInfo,
               host,
               RollingUpdateError.IMAGE_MISSING,
               metadata);
     }
     if (taskStatus.getThrottled().equals(ThrottleState.IMAGE_PULL_FAILED)) {
       return opFactory.error(
-              "timed out waiting for job to reach RUNNING due to failure pulling Docker image " +
-                      String.format("(previous job states: %s)", previousJobStatesString),
+              baseError + "due to failure pulling Docker image " + stateInfo,
               host,
               RollingUpdateError.IMAGE_PULL_FAILED,
               metadata);
     }
-    if (taskStatus.getState().equals(TaskStatus.State.HEALTHCHECKING)) {
```

Comment:
We'll indicate that the terminal state was `HEALTHCHECKING` in `stateInfo`. Possibly makes the errors a little less readable, but it will shed light on whatever the final state was without needing a switch or a bunch of conditionals.